darwin/cli_functions.py

```diff
@@ -652,18 +654,24 @@ def set_file_status(dataset_slug: str, status: str, files: List[str]) -> None:
         _error(f"No dataset with name '{e.name}'")
 
 
-def delete_files(dataset_slug: str, files: List[str]) -> None:
+def delete_files(dataset_slug: str, files: List[str], yes: bool = False) -> None:
```

Comment:
While I think it's a good idea to call this argument `yes` in the parser, I think this should be named differently here. Something like `skip_confirmation`?

Reply:
Done !